nipype/interfaces/cmtk/convert.py

```diff
@@ -194,17 +194,17 @@ def _run_interface(self, runtime):
             for data in self.inputs.data_files:
                 _, data_name, _ = split_filename(data)
                 cda = cf.CData(name=data_name, src=data, fileformat="NumPy")
-                if not string.find(data_name, "lengths") == -1:
+                if string.find(data_name, 'lengths') != -1:
```

Comment:
`string.find` hasn't existed since Python 2.7...

```suggestion
                if 'lengths' in data_name:
```